hyperscale/005-create_postgis_extension.sql

```diff
@@ -0,0 +1,26 @@
+----------------------------------------------------------------------------------
```

Comment:
Why do we need this file if we are sure that all the contents will be commented out?

Reply:
I keep it in case developers may want to use this file to enable it

Reply:
I think they can easily add additional lines to hyperscale/006-other_extensions.sql, I'd prefer removing this one.

Reply:
Let's remove this file. We may even need to renumber all the files `006-other_extensions.sql` to `010-stat_statements_reset.sql` after removing this one.

Reply:
Removed